src/main/resources/doc/en/html/libs/gates/controlled.html

```diff
@@ -74,86 +74,190 @@ <h2>
+              <td class="unov">
+                1
+              </td>
+              <td class="zerov">
+                0
+              </td>
+            </tr>
+			<tr>
+              <td class="tspace">
+                <b class="uvalue">U</b>/<b class="evalue">E</b>
+              </td>
+              <td class="tunov">
+                1
+              </td>
+              <td class="evalue">
+                E
+              </td>
+              <td class="evalue">
+                E
+              </td>
+			</tr>
+			<tr>
+              <td class="tspace">
+                All
+              </td>
+              <td class="tspace">
+                <b class="uvalue">U</b>/<b class="evalue">E</b>
+              </td>
+              <td class="evalue">
+                E
+              </td>
+              <td class="evalue">
+                E
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </center>
+	  </p>
       <p>
         Controlled buffers can be useful when you have a wire (often called a <em>bus</em>) whose value should match the output of one of several components. By placing a controlled buffer between each component output and the bus, you can control whether that component's output is fed onto the bus or not.
       </p>
       <h2>
         Pins (assuming component faces east, control line right-handed)
       </h2>
+	  <div class="attliste">
       <dl>
-        <dt>
-          West edge (input, bit width matches Data Bits attribute)
+        <dt class="lib">
+          West edge:
         </dt>
         <dd>
-          The component input that will be used to compute the output if the control input is 1.
+          The component input that will be used to compute the output if the control input is <b class="unov">1</b>. Bit width matches  <b class="propertie">Data Bits</b> attribute.
+        </dd>
+        <dt class="lib">
+          South edge:
         </dd>
-        <dt>
-          South edge (input, bit width 1)
-        </dt>
         <dd>
-          The component's control input.
+          The component's control input. Bit width 1.
         </dd>
-        <dt>
-          East edge (output, bit width matches Data Bits attribute)
+        <dt class="lib">
+          East edge:
         </dt>
         <dd>
-          The component's output, which will be floating if the control input is 0 or floating, the error value if the control input is the error value, and will be computed based on the west-side input if the control input is 1.
+          The component's output, which will be floating (<b class="uvalue">U</b>) if the control input is <b class="zerov">0</b> or floating (<b class="uvalue">U</b>), the error value (<b class="evalue">E</b>) if the control input is the error value (<b class="evalue">E</b>), and will be computed based on the west-side input if the control input is <b class="unov">1</b>.  Bit width matches  <b class="propertie">Data Bits</b> attribute.

```

Comment:
"Bit width matches  <b class="propertie">Data Bits</b> attribute." -> "The bit width matches  <b class="propertie">Data Bits</b> attribute."

PS: There's a typo in the class name: "propertie" -> "property". Maybe this would need fixing in more files. Then, this should be done in a separate PR.

Reply:
Indeed, it's a style name that I messed up a bit.
I will correct it but it will impact (195) many files.
But it works.